backend/src/main/java/com/sim_backend/Main.java

```diff
@@ -33,6 +33,6 @@ public static void main(final String[] args) {
         }).start(backendPort); // Start the server
 
         // Register a test route
-        TestMessageController.registerRoutes(app);
+        //TestMessageController.registerRoutes(app);
```

Comment:
Was commenting this out intentional?

Reply:
I meant to revert this, I just needed a change in Main.java to make sure the workflow is actually picking up on the changed files.